src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1224,9 +1209,13 @@ TxQ::apply(
     if (!replacedTxIter && isFull())
     {
         auto lastRIter = byFee_.rbegin();
-        if (lastRIter->account == account)
+        while (lastRIter != byFee_.rend() && lastRIter->account == account)
         {
-            JLOG(j_.warn())
+            ++lastRIter;
+        }
+        if (lastRIter == byFee_.rend())
```

Comment:
Good point. It does happen in a unit test where the limits are much smaller, and I suppose it would be possible if someone set their config to some unual values.

Reply:
Good point. It does happen in a unit test where the limits are much smaller, and I suppose it would be possible if someone set their config to some unual values.

Reply:
Done